astroplan/core.py

```diff
@@ -25,8 +40,10 @@ class Observer(object):
     Some comments.
     """
 
-    def __init__(self, name, longitude, latitude, elevation, pressure,
-                 relative_humidity, temperature, timezone, description=None):
+    def __init__(self, name=None, location=None, latitude=None, longitude=None,
```

Comment:
Should we instead use the `**kwargs` syntax and then `kwargs.get` or `kwargs.pop` the keywords one at a time?